### DISCUSSION TITLE
任務導向學習支援系統-AMIGOS

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#176] [General]
**新聞內容：**
美國國防高等研究計畫署近日與富士全錄、羅斯托克大學、加州大學聖塔芭芭拉分校和 AR 公司 Patched Reality 組成的團隊，簽署 5,800 萬美元合約，將開發頭戴式 AI 與 AR 技術、即時協助醫護兵和工程師動手術或維修器材的設備。
名為自動多模式任務導向學習支援系統（AMIGOS）的裝置由兩設備組成，第一部分是機械視覺與機械學習的**資料庫系統**，掃描儲存大量醫學、機械設備說明、圖像、影片等資料。
另一部分是由 **AI 驅動的頭戴式裝置**，透過 AR 技術分析醫護兵或工程師面對的醫療手術或機械損壞部分，和資料庫比對分析，並即時指引下一個步驟如何操作，讓使用者即使沒遇過類似狀況，也可一步步照 AI 指示處理。
雖然 AMIGOS 不能完全取代訓練有素的醫生或工程師，但戰爭期間很多情況下這些專家也無法隨傳隨到，因此設備期望藉由 AI 技術，讓即使沒有相關訓練的人員也能緊急簡單醫護和維修，大幅降低後勤負擔和人員裝備損失率。
**新聞心得：**
雖然這項系統花費的金錢有夠龐大，但是很值得！讓無醫療專業的人透過分析過後的指示處理狀況，不僅能先讓運送回來的人員或是器具做及時的維修，更不必擔心需要臨時調動多少人力去協助支援。
**新聞來源：**
https://technews.tw/2022/01/04/darpa-is-developing-amigos-which-can-help-medics-and-mechanics-operate-unfamiliar-procedures-with-ai-and-ar-technologies/
![size0-full-624x415](https://user-images.githubusercontent.com/90883526/148309067-04d8371c-5ead-4800-9d89-f31f0bd6c72b.jpeg)

**Comment by briankuo123** [comment]
還蠻有趣的，我有聽說過AI在醫療方面的貢獻，但沒聽說過有人拿AI去分析損壞的機械。想必以後自己家裡有東西壞掉也可以靠AI的指示去槍它修好。

**Comment by wh20502** [comment]
以後會不會買東西附的操作說明，都會改成附ar眼鏡帶你邊用邊學。
